translations/src/config.ts

```diff
@@ -127,7 +127,11 @@ class Config {
     fa_pr: ['pes'],
     'de-si': ['de'],
     sr: ['sr-Cyrl'],
-    'zh-hans': ['zh-CN']
+    'zh-hans': ['zh-CN'],
```

Comment:
so `zh-hans` is no longer "supported"? Can we be sure it won't be needed in the now or in the future?